sda/service_test.go

```diff
@@ -126,6 +150,45 @@ func TestServiceNew(t *testing.T) {
 	waitOrFatal(ds.link, t)
 }
 
+func TestServiceChannels(t *testing.T) {
+	sc1 := &ServiceChannels{}
+	sc2 := &ServiceChannels{}
+	var count int
+	// Needed because of TestServiceNew which listens on a channel. Strange that
+	// it worked before.
+	DeleteNewService("DummyService")
```

Comment:
Maybe, I don't really get that: If `TestServiceNew` creates a "DummyServices" it should be `TestServiceNew` which does its cleanup and calls `DeleteNewService("DummyService")` or not?